lib/testLauncher/SuitestLauncher.js

```diff
@@ -28,7 +29,11 @@ class SuitestLauncher {
 		log.createTestPack();
 		validateInput('AUTOMATED', this.ownArgv);
 		try {
-			const {tokenKey, tokenPassword, testPackId, logDir} = this.ownArgv;
+			const {tokenKey, tokenPassword, testPackId, logDir, inspect, inspectBrk} = this.ownArgv;
+
+			if (inspect || inspectBrk) {
+				throw new Error(texts['tl.inspectOnlyForInteractiveMode']());
```

Comment:
I think SuitestError should be used, perhaps of INVALID_INPUT type.

Reply:
In case of SuitestError output in console will be rather big with call stack, not sure that it has sense to show this info to user, If you do not like using just Error probably it is better to rewrite it to avoid using Error at all?

Reply:
I think when we throw Error explicitly it should be SuitestError, so its clear that it was defined by our library. (AssertionError for assertions). In this case if using inspect || inspectBrk doesn't break anything, maybe its better to just display a warning with logger.warn? @andriichumak